frontend/src/components/custom-tools/prompt-card/PromptCardItems.jsx

```diff
@@ -184,103 +353,235 @@ function PromptCardItems({
+                          profileId === selectedLlmProfileId && index !== 0
+                            ? -10
+                            : 0,
+                      }}
+                      transition={{ duration: 0.5, ease: "linear" }}
+                      className="prompt-card-llm"
+                    >
+                      <Col
+                        key={profileId}
+                        className="prompt-card-llm-container"
+                        xs={{ span: getColSpan() }}
+                      >
+                        <Divider className="prompt-card-divider" />
+                        <Space
+                          direction="vertical"
+                          className="prompt-card-llm-layout"
+                        >
+                          <div className="llm-info">
+                            <Image
+                              src={profile?.icon}
+                              width={15}
+                              height={15}
+                              preview={false}
+                              className="prompt-card-llm-icon"
+                            />
+                            <Typography.Title
+                              className="prompt-card-llm-title"
+                              level={5}
+                            >
+                              {profile?.conf.llm}
+                            </Typography.Title>
+                          </div>
+                          <div className="prompt-cost">
+                            <Typography.Text className="prompt-cost-item">
+                              Tokens:{" "}
+                              {!singlePassExtractMode && (
+                                <TokenUsage tokenUsageId={tokenUsageId} />
+                              )}
+                            </Typography.Text>
+                            <Typography.Text className="prompt-cost-item">
+                              Time: {timers[tokenUsageId] || 0}s
+                            </Typography.Text>
+                            <Typography.Text className="prompt-cost-item">
+                              Cost: ${getFormattedTotalCost(result, profile)}
+                            </Typography.Text>
+                          </div>
+                          <div className="prompt-info">
+                            <CheckableTag
+                              checked={enabledProfiles.includes(profileId)}
+                              onChange={(checked) =>
+                                handleTagChange(checked, profileId)
+                              }
+                              style={{
+                                backgroundColor: checked
+                                  ? "#F6FFED"
+                                  : "#00000005",
+                                borderColor: checked ? "#B7EB8F" : "#00000026",
+                                color: checked ? "#52C41A" : "#000",
+                              }}
```

Comment:
@jagadeeswaran-zipstack Can this styling be moved to css file? Please make sure we follow the same everywhere.

Reply:
Sure